src/Bootstrapper/Navigation.php

```diff
@@ -82,8 +82,13 @@ class Navigation extends RenderedObject
      * @var bool Whether the navigation links float right or not
      */
     protected $right = false;
+	
+	/**
+     * @var bool Whether the navigation links float left or not
+     */
+	protected $left = false;
```

Comment:
I think this introduces a bit too much redundancy. You can either have `navbar-left` or `navbar-right` not both so there's no point having two class vars. It'd be better if you renamed the `$right` variable to `$float`, updated the comment and the implementation of `right()` and `left()`.